k8s/openebs-monitoring-pg.yaml

```diff
@@ -0,0 +1,180 @@
+# The following file is intended for deployments that are not already
+# configured with prometheus. This is a minified version of the config
+# from the files present under ./openebs-monitoring/
+# 
+# Prometheus tunables
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: openebs-prometheus-tunables
+data:
+  storage-retention: 24h
+---
+# Define the openebs prometheus jobs
+kind: ConfigMap
+metadata:
+  name: openebs-prometheus-config
+apiVersion: v1
+data:
+  prometheus.yml: |-
```

Comment:
Is there any controller or logic who understand this data ?

Reply:
I guess prom/prometheus:v1.7.2 understands this ?

Reply:
yes. the prometheus container understands it, it is passed as a volume mount.